torch/ao/nn/quantizable/modules/rnn.py

```diff
@@ -83,6 +83,7 @@ def forward(self, x: Tensor, hidden: Optional[Tuple[Tensor, Tensor]] = None) ->
         fgate_cx_igate_cgate = self.fgate_cx_igate_cgate.add(fgate_cx, igate_cgate)
         cy = fgate_cx_igate_cgate
 
+        # TODO: make this tanh a member of the module so its qparams can be configured
```

Comment:
do we need tanh qparams to be configurable for the current use case?

Reply:
Not sure. I tried to keep changes in this file separate for now. If it turns out it's needed I can submit a follow up PR

Reply:
Not sure. In this PR I'm trying to isolate the changes to within my util functions. If we need to configure this we can fix this separately in the future.